ts/screens/wallet/TransactionDetailsScreen.tsx

```diff
@@ -59,7 +98,7 @@ export class TransactionDetailsScreen extends React.Component<Props, never> {
 
   public render(): React.ReactNode {
     const { navigate } = this.props.navigation;
-    const operation: WalletTransaction = this.props.navigation.state.params
+    const transaction: WalletTransaction = this.props.navigation.state.params
```

Comment:
I suggest to move the current `WalletTransaction` to a property of this component and implement a redux connector that extracts it from the navigation state - see an example [here](https://github.com/teamdigitale/italia-app/blob/master/ts/screens/onboarding/PinScreen.tsx#L273) - @fpersico do you agree?

Reply:
A selector is always good to avoid changing code everywhere if the state shape changes.
The problem i see here is different (don't know why i didn't spot it before): why we are passing the whole transaction object and not just an id? This for sure will not work (or not in the default way) with deeplinking.

Reply:
Will push a (pretty large) commit tomorrow, I may have gotten carried away and started implementing things 😅